### PR TITLE
Feature/dit 168/work instructions users

### PR DESCRIPTION
**Pull request by lencodes**

### CODE REVIEW

File: documentation/work-instructions/user-management.md [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+   2. Email*: the email address of the new user
+   3. Organization*: the organization to which the new user should belong
+   4. Applications*: one or more applications to which the new user has access to
+   5. Password*: set an initial password for the user (retype it in the second field for verification)
+
+3. Optionally fill in the _optional_ fields
+   1. Description: optionally set a description of the user
+   2. Locale: optionally set the locale of the user (e.g. en-US)
+   3. Person: optionally add extra information about the person
+   4. Security Groups: optionally add Security Groups to this user to give them more rights
```
Comment (remko48) [inline documentation/work-instructions/user-management.md:32]: Security Groups is also a required field.

File: documentation/work-instructions/user-management.md [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+## Step 3: Editing an existing user
+
+1. Click on "-> Details" to navigate to the detail page of the to edit user
+
+2. Edit one or multiple fields in the form
+
+3. Click on "Save" to complete the process
+
+> Tip: click on "+ Add to dashboard" or "- Remove from dashboard" to toggle users as Dashboard Cards on the dashboard page!
+
```
Comment (remko48) [inline documentation/work-instructions/user-management.md:48]: Maybe we should add an extra tip for the password fields:
`Password wil not be reset when saving the user if the fields are empty.`

File: documentation/work-instructions/user-management.md [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+### 4.2 Deleting multiple users
+
+1. Check all rows of all the to be removed users
+
+2. Click on "Bulk actions"
+
+3. Click on "Delete"
+
+4. Confirm the deltion by clicking "Delete" to complete the process
```
Comment (remko48) [inline documentation/work-instructions/user-management.md:71]: deltion -> deletion

**Review by remko48 — CHANGES_REQUESTED** [review]
Nice instructions 👍

**Review by remko48 — APPROVED** [review]
Looks good.

**Review by WilcoLouwerse — APPROVED** [review]

**Pull request merged**